services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/store/device/TableManagementStore.java

```diff
@@ -887,44 +922,60 @@ private List<CommonCredential> parseCredentials(final ResultSet result) {
     /**
      * Gets a list of devices of a specific tenant.
      *
-     * @param tenantId the tenantId to search devices
-     * @param pageSize the page size
-     * @param pageOffset the page offset
+     * @param tenantId    The tenantId to search devices.
+     * @param pageSize    The page size.
+     * @param pageOffset  The page offset.
+     * @param filters     The list of filters.
```

Comment:
FMPOV we should document that only the first item in the list will be considered.

Reply:
I change the "The list of filters." to "The list of filters (currently only the first value of the list is used)."